README.md

```diff
@@ -2,23 +2,163 @@
 
 A minimal GameCube IPL
 
+IPLBoot will attempt to load DOLs from the following locations in order:
+- USB Gecko in Card Slot B
+- SD Gecko in Card Slot B
+- USB Gecko in Card Slot A
+- SD Gecko in Card Slot A
+- SD2SP2
+
+When loading from SD Gecko or SD2SP2, it will look for and load different filenames depending on what buttons are being held at boot. You can use this to change the boot behavior by holding a button on your controller. 
+
+ Button Held | File Loaded
+-------------|--------------
+ *None*      | `/ipl.dol`
+ A           | `/a.dol`
+ B           | `/b.dol`
+ X           | `/x.dol`
+ Y           | `/y.dol`
+ Z           | `/z.dol`
+ Start       | `/start.dol`
+ D-Left      | `/left.dol`
+ D-Right     | `/right.dol`
+ D-Up        | `/up.dol`
+
+If no DOL can be loaded, IPLBoot will boot the official IPL (GameCube intro and menu) instead.
+
+For example, this configuration would boot straight into Swiss by default, or GBI if you held B, or the typical GameCube intro if you held any other button:
+- `/ipl.dol` - Swiss
+- `/b.dol` - GBI
+
+This configuration would boot into the typical GameCube intro by default, or Swiss if you held Z, or GBI if you held B:
+- `/z.dol` - Swiss
+- `/b.dol` - GBI
+
+**Pro-tip:** You can prevent files from showing in Swiss by marking them as hidden files on the SD card.
+
+If you hold multiple buttons, the highest in the table takes priority. Also note that all directions are D-Pad buttons, *not* joysticks. Careful not to touch any of the analog controls (sticks and triggers) when powering on as this is when they are calibrated.
+
+**Something not working?** See the [troubleshooting section](#troubleshooting).
+
+
 ## Usage
 
-Flash the latest build to your Qoob Pro or Viper as a BIOS.  
-The Qoob SX is currently not supported because it uses a very different process
-to boot, and reverse-engineering efforts so far have been unsuccessful.
+Download the designated appropriate file for your device from the [latest release](https://github.com/redolution/iplboot/releases/latest).
+
+Prepare your SD card by copying DOLs onto the SD card and renaming them according the table above.
+
+### PicoBoot
+
+You can download ready-to-go firmware featuring IPLBoot directly from PicoBoot. Just follow the [installation guide](https://github.com/webhdx/PicoBoot#-installation-guide). This is likely what you want.
+
+Otherwise, use `iplboot.dol` and follow the PicoBoot [compiling firmware](https://github.com/webhdx/PicoBoot#compiling-firmware) instructions to flash your Raspberry Pico.
+
+### Qoob Pro
+
+Use `iplboot.gcb` to flash your Qoob Pro as a BIOS.
+
+### Qoob SX
+
+Qoob SX is not currently supported.
```

Comment:
How about?

```suggestion
Qoob SX is not currently supported due to size constraints.

The last version available for the Qoob SX is [r5.2](https://github.com/redolution/iplboot/releases/tag/r5.2). Download [`qoob_sx_iplboot_upgrade.elf`](https://github.com/redolution/iplboot/releases/download/r5.2/qoob_sx_iplboot_upgrade.elf) and flash it to your Qoob SX as a BIOS.
```

Reply:
I'd rather not link directly to files. The link to the release is enough.

The other thing is that installing to the Qoob SX is a bit more involved than this. If I recall correctly, you need to burn the file to an ISO 9660 disc, and turn on the GC while holding left on the dpad, which will cause the Qoob to boot the fallback BIOS. Then you can run the updater from there.
There are instructions in the Qoob readme, maybe we should copy them here.

Reply:
Are those instructions available anywhere online we could link to? Otherwise we can probably put the onus of knowing how to use/flash their device on the user.

Reply:
Other than the archived files on the gcf forum page I linked, I don't think there is a reasonable source that's still online. I prefer copying the instructions here for ease of access.

Reply:
Updated link. I'll let you cover those instructions then as I have no experience with the device. The NFO file from the Qoob 1.3c release sates:

> This release includes a network / dvd updater. Just run the qoob_13c_upgrade.elf somehow (either by burning as iso9660 to a dvd or by uploading per network).
> 
> If you burn it to a cd, make sure to fill up the dvd to at least 1.2GB of data else the gc drive can't read it.
> 
> ...
> 
> YOU CAN *NOT* BREAK YOUR SX BY FLASHING WRONG FIRMWARE INTO IT.
> 
> The SX has an integrated "backup" bios. You can go into it by holding the D-Pad to the left (D-pad is the small digital pad at the lower left of the controller). Do not boot the disc, you also have to press 'B'.
> 
> You *need* to start the upgrader from this bios. This is done due to protect you from evil applications which could otherwise erase your bios. Still, the backup bios is read only and can't be erased.
> 
> So there's absolutely no way to kill your SX. If you flash something else to it, all it would do is to stop booting until you boot into the recovery bios.
> 
> To summarize it:
> 
>  - Don't panic. You can't break your SX by flashing.
>  - You can't flash .GCB files, you need the .ELF updaters.
>  - To flash via DVD, you need to hold "D-Pad-Left" while booting. Be sure to pad your disc to at least ~1GB of data!
>  - To flash via Network, you need to hold "D-Pad-Left"+"B" while booting. IP address will be 192.168.1.32. There's no way to change that.

Reply:
Alright, leave the instructions out for now. I'll take care of that later.